### ISSUE TITLE
Failed Periphery install via script on Alpine

### ISSUE THREAD

**Issue by Nonobis** [moghtech/komodo#66]
Hello,

First great project and beautiful .. hope i can now at least drop portainer :)

i have found an bug in trying to install "Periphery " with the script, the script is not working on alpine

Environnement :

LXC container en proxmox based on Alpine (created with Proxmox Ve Helper)

Error :

```
bagetter:/tmp# komodo
-sh: komodo: not found
bagetter:/tmp# komodosystemctl status periphery
-sh: komodosystemctl: not found
bagetter:/tmp# systemctl status periphery
-sh: systemctl: not found
bagetter:/tmp# curl -sSL https://raw.githubusercontent.com/mbecker20/komodo/main/scripts/setup-periphery.py | python3
=====================
 PERIPHERY INSTALLER
=====================
version: v1.14.0
user install: False
bin dir: /usr/local/bin
config dir: /etc/komodo
service file dir: /etc/systemd/system
/bin/sh: systemctl: not found

config already exists, skipping...
service file already exists, skipping...
starting periphery...
/bin/sh: systemctl: not found

Finished periphery setup.

Note. Use "systemctl status periphery" to make sure periphery is running
Note. Use "systemctl enable periphery" to have periphery start on system boot

-sh: bagetter:/tmp#: not found
bagetter:/tmp# -sh: komodosystemctl: not found
-sh: -sh:: not found
bagetter:/tmp# bagetter:/tmp# systemctl status periphery
-sh: bagetter:/tmp#: not found
bagetter:/tmp# -sh: systemctl: not found
-sh: -sh:: not found



```

**Comment by mbecker20** [comment]
Right, the installer requires systemd, which is included in Debian, fedora, Ubuntu, opensuse, and many other distro. But not alpine. The periphery container should work though, see [the docs](https://komo.do/docs/connecting-servers#install-the-periphery-agent---container)](https://komo.do/docs/connecting-servers#install-the-periphery-agent---container).

Or you can download the periphery binary from the latest release, and use the process manager alpine uses instead of systemd. Not too familiar with alpine for more specifics though.

**Comment by mbecker20** [comment]
Maybe someone would be familiar with packaging the binary for alpine distribution.

**Comment by borestad** [comment]
FYI: The binary (v1.17) doesn't work with alpine anyway because of musl and not libc. It needs it's own binary to work.

`Error relocating /usr/local/bin/periphery: __res_init: symbol not found`

Unfortunately, the docker image ghcr.io/moghtech/komodo-periphery is 432Mb, so the pro's of using a slim alpine build doesn't really benefit from using such a large image anyway.

All in all - two things could be improved:
* Create a musl artifact for periphery
* Slim down the container.

**Comment by mbecker20** [comment]
Just as a final note, at this point I only provide x86_64 and aarch64 libc compatible binaries and images. Alpine / musl users can consider forming together for a community distribution to handle this platform, or compile the binaries from source themselves.

**Comment by nano9g** [comment]
I’ve set up a [repository](https://github.com/nano9g/komodo-alpine/) that automatically generates Alpine builds of Periphery. It’s worked well for me over the past two releases, so I think it’s ready to share. Hopefully others will find it useful!
